src/components/reusables/Toggle.tsx

```diff
@@ -33,7 +32,13 @@ export default class Toggle extends React.Component<ToggleOwnProps, ToggleState>
       <form className="toggle-container">
         { label && <label className={this.state.on ? "active" : ""}><span>{label}</span></label> }
         <div className={`toggle${this.state.on ? " toggle-on" : ""}`}>
-          <button tabIndex={0} className={`slider${this.state.on ? " slider-on" : ""}`} aria-pressed={this.state.on} onClick={this.toggle} />
+          <button
+            tabIndex={0}
+            className={`slider${this.state.on ? " slider-on" : ""}`}
+            aria-label="Toggle button"
```

Comment:
Buttons need text and these don't have it, which is fine, but should have an aria-label.